guides/webops_community_of_practice.md

```diff
@@ -0,0 +1,21 @@
+---
+category: Communities of Practice
+expires: 2020-06-01
+---
+# WebOps Community of Practice
+
+## Manifesto
+
+- TBC
+
+## Charter
+
+- TBC
+
+## Membership
+
+Meetings of the community are every Monday in Birmingham. All OPG WebOps and interested parties (Technical Architects, Heads of Profession, Developers, Product Managers and so on) are welcome.
+
+## Related Resources
+
+[Community Trello Board](https://trello.com/b/NgygrWKD/webops-community-backlog)
```

Comment:
Should we make this board public?

Reply:
That's a good question. Depends what info is there, I suppose. It's worth linking even if not public as people can ask for access.

Reply:
Yeah go for it :)